src/lib.rs

```diff
@@ -36,6 +36,10 @@ pub use parser::DocError;
 pub use parser::DocParser;
 pub use printer::DocPrinter;
 
+#[cfg(feature = "wee_alloc")]
+#[global_allocator]
+static ALLOC: wee_alloc::WeeAlloc = wee_alloc::WeeAlloc::INIT;
```

Comment:
@lucacasonato claims that it's not really necessary - there's not much of a difference between default and `WeeAlloc`. I don't have a strong opinion either way.

Reply:
I will remove it for now in a follow-up PR.  It saves about ~8k, but the wasm is a lot bigger for other reasons that 8k is a drop in the bucket and it can impact performance.